duties/constants/program.py

```diff
@@ -9,6 +9,7 @@
 REQUEST_TIMEOUT = (3, 5)
 REQUEST_CONNECTION_ERROR_WAITING_TIME = 2
 REQUEST_READ_TIMEOUT_ERROR_WAITING_TIME = 5
+REQUEST_HEADER = {"Content-type": "application/json", "Accept": "text/plain"}
```

Comment:
the response should be json as well, I don't think any client returns plain text

```suggestion
REQUEST_HEADER = {"Content-type": "application/json", "Accept": "application/json"}
```